packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -766,6 +766,15 @@ describe('Joy <Autocomplete />', () => {
       });
     });
 
+    it('should open popup when clicked on borders of root element', () => {
+      const handleOpen = spy();
+      render(<Autocomplete onOpen={handleOpen} options={['one']} />);
```

Comment:
Render `TextField` with `renderInput` prop.

Reply:
In Joy UI, you don't need to render a TextField. The Autocomplete is basically an input.